### PR TITLE
Responsiveness for marketplace and feedback pages

### PR DESCRIPTION
**Pull request by Funmi7**
##  What does this PR do
Responsiveness for marketplace and feedback pages

##  Description of task to be completed
Makes the marketplace and feedback responsive for different screen sizes

##  How should this manually be tested
npm start the project and check with different screen sizes

##  What are the relevant Trello board stories
Responsive Design Marketplacepage
Feedback page responsive design

##  Questions (If any):

### CODE REVIEW

File: src/views/Marketplace/Marketplace.js [hunk 1]
```diff
@@ -40,6 +46,10 @@ const StyledMarketplace = styled.div`
     width: 100%;
     justify-content: space-evenly;
     padding-top: 1rem;
+
+    /* @media ${devices.mobile} {
+      width: 80%;
+    } */
```
Comment (BenjaminGrabow) [inline src/views/Marketplace/Marketplace.js]: Let's delete all comments :)

File: src/views/Feedback/Feedback.js [hunk 2]
```diff
@@ -89,10 +115,13 @@ const Feedback = ({ user, getFeedback, feedback }) => {
...
-          <ChartCardContainer>
-            <StudentChart />
-          </ChartCardContainer>
-          <StyledFeedback className='feedback-card-container'>
+          <div className='chart-display'>
+            <ChartCardContainer>
+              <StudentChart />
+            </ChartCardContainer>
+          </div>
+          {/* <StyledFeedback */}
```
Comment (BenjaminGrabow) [inline src/views/Feedback/Feedback.js]: Here is also one. I will merge this PR after you made the requested changes. Great work Funmi!

**Comment by Funmi7** [comment]
[Responsive Design Marketplacepage](https://trello.com/c/L52hNQLZ/118-responsive-design-marketplacepage)

**Comment by Funmi7** [comment]
[Responsive Design Marketplacepage](https://trello.com/c/L52hNQLZ/118-responsive-design-marketplacepage)

**Pull request merged**